var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -256,6 +256,11 @@ def cmake_args(self):
                 '-DGMX_MPI:BOOL=OFF',
                 '-DGMX_THREAD_MPI:BOOL=ON'])
 
+        if self.spec.satisfies('%aocc'):
+            opt_flags = '-Ofast -ffast-math'
+            options.append(self.define('CMAKE_C_FLAGS_RELEASE', opt_flags))
```

Comment:
Note that the default build type is RelWithDebInfo, so probably that should get the those flags, too.

Reply:
Ah ok, did'nt know that!

Reply:
Do you really need -ffast-math? It's know to lead to all sorts of problems. https://simonbyrne.github.io/notes/fastmath/

Reply:
I agree, just use `cflags=...` for special OMG fast builds.